apps/mobile/src/components/register-adoption-form/adoption-form.component.test.tsx

```diff
@@ -0,0 +1,140 @@
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
+
+import RegisterAdoptionForm from './adoption-form.component';
+import { AdoptionSteps } from './adoption-form.types';
+import Routes from '../../routes';
+import Home from '../../screens/home/home.screen';
+import { renderWithProviders, screen, fireEvent, act, waitFor } from '../../test/test-utils';
+
+const mockShow = jest.fn();
+jest.mock('native-base', () => ({
+  ...jest.requireActual('native-base'),
+  useToast: () => ({
+    show: mockShow,
+    isActive: () => false,
+  }),
+}));
+
+const forwardStep = () => {
+  fireEvent.press(screen.getByText(/pr[oó]xima etapa/gi));
+};
+
+const fillOutTextInput = async (testId: string, value: string) => {
+  const input = await screen.findByTestId(`adoption-form-${testId}-input`);
+  fireEvent.changeText(input, value);
+};
+
+const pickOptionFromList = async (optionText: string | RegExp) => {
+  const option = await screen.findByText(optionText);
+  fireEvent.press(option);
+};
+
+const goToLastStep = async () => {
+  await fillOutTextInput('name', 'Bob');
+  forwardStep();
+  await fillOutTextInput('breed', 'pitbull');
+  forwardStep();
+  await pickOptionFromList(/c[aã]o/gi);
+  forwardStep();
+  await screen.findByTestId(`adoption-form-age-input`);
```

Comment:
Wouldn't be better to find it by accessibilityLabel or something else? I'd suggest always relying on accessibility especially because it's an open-source project with great visibility.

Reply:
Yes, for sure. However, in this specific case, I don't how to get the element via an accessibility label since it's a slider component.